app/api/os-places.js

```diff
@@ -0,0 +1,103 @@
+const config = require('../config')
+const wreck = require('@hapi/wreck')
+const { mapOsCountryCodeToCountry, titleCase } = require('../lib/format-helpers')
+
+const baseUrl = config.osPlacesApi.baseUrl
+const postcodeEndpoint = 'postcode'
+
+const options = {
+  headers: {
+    key: config.osPlacesApi.token
+  },
+  json: true
+}
+
+const getPostcodeAddresses = async (postcode, houseNumber) => {
```

Comment:
Would it be too much to add jsdoc to this, would be helpful for future, especially with the nested arrays